### PR TITLE
Blog post meetup

### PR DESCRIPTION
**Pull request by balloob**
**Description:**
Blog post about the meetup.

Still waiting for the correct video.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_posts/2018-11-16-state-of-the-union.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/MGEz2iB6m94" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Special thanks to:
+
+ - Jeroen Joosse for the initiative and organizing
+ - Erik-Jan Riemers for co-organizing and making it this big
+ - ING for hosting the event
+ - Franck Nijhof for providing the Hass.io demos
+ - Nicolaas Schoute for providing the tensorlfow demos
```
Comment (fabaff) [inline source/_posts/2018-11-16-state-of-the-union.markdown]: `tensorlfow` -> `TensorFlow`

**Comment by riemers** [comment]
lgtm ;p

File: source/_posts/2018-11-16-state-of-the-union.markdown (hunk 1 above)
Comment (frenck) [reply source/_posts/2018-11-16-state-of-the-union.markdown]: Updated, thx.

File: source/_posts/2018-11-16-state-of-the-union.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/egcOCWIh9jQ" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Special thanks to:
+
+ - Jeroen Joosse for the initiative and organizing
+ - Erik-Jan Riemers for co-organizing and making it this big
+ - ING for hosting the event
+ - Franck Nijhof for providing the Hass.io demos
+ - Nicolaas Schoute for providing the TensorFlow demos
```
Comment (klaasnicolaas) [inline source/_posts/2018-11-16-state-of-the-union.markdown:30]:
```suggestion
 - Klaas Schoute for providing the TensorFlow demos
```

**Review by frenck — APPROVED** [review]

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
🎉

**Pull request merged**